src/sentry/relay/config/__init__.py

```diff
@@ -138,10 +137,13 @@ def get_project_config(project, full_config=True, project_keys=None):
 
     :return: a ProjectConfig object for the given project
     """
-    metrics.incr("sentry.relay.config.get_project_config")
-    with configure_scope() as scope:
+    with sentry_sdk.push_scope() as scope:
         scope.set_tag("project", project.id)
+        with metrics.timer("relay.config.get_project_config.duration"):
+            return _get_project_config(project, full_config=True, project_keys=None)
```

Comment:
Could you use `@metrics.wraps("relay.config.get_project_config.duration")` on the `get_project_config function instead?